src/server/scripts/EasternKingdoms/OperationGnomeregan/operation_gnomeregan.cpp

```diff
@@ -0,0 +1,2565 @@
+            }
+
+            void HandleDecorByPhase(Creature* /*me*/, int8 /*p*/) {
+                /*
+                switch (p)
+                {
+                case 1: // INTRO
+                    if (decor_p1 == true) break;
+                    // 8 Gnomeregan banners
+                    for (int8 n = 0; n < 8; ++n)
+                        me->SummonGameObject(GO_BANNER, decor_gnomeregan_banners[n].m_positionX, decor_gnomeregan_banners[n].m_positionY, decor_gnomeregan_banners[n].m_positionZ, decor_gnomeregan_banners[n].m_orientation, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0);
+
+                    decor_p1 = true;
+                    break;
+
+                case 2: // AIRFIELD
+                    if (decor_p2 == true) break;
+                    // 5 Gnomeregan banners
+                    for (int8 n = 8; n < (n + 5); ++n)
+                        me->SummonGameObject(GO_BANNER, decor_gnomeregan_banners[n].m_positionX, decor_gnomeregan_banners[n].m_positionY, decor_gnomeregan_banners[n].m_positionZ, decor_gnomeregan_banners[n].m_orientation, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0);
+
+                    decor_p2 = true;
+                    break;
+
+                case 3: // SMALL HOUSE (RADIATION VALVE)
+                    if (decor_p3 == true) break;
+                    // 5 Gnomeregan banners
+                    for (int8 n = 13; n < (n + 5); ++n)
+                        me->SummonGameObject(GO_BANNER, decor_gnomeregan_banners[n].m_positionX, decor_gnomeregan_banners[n].m_positionY, decor_gnomeregan_banners[n].m_positionZ, decor_gnomeregan_banners[n].m_orientation, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0);
+
+                    decor_p3 = true;
+                    break;
+
+                case 4: // OUTSIDE CAVE OF GNOMEREGAN
+                    if (decor_p4 == true) break;
+                    // 5 Gnomeregan banners
+                    for (int8 n = 18; n < (n + 5); ++n)
+                        me->SummonGameObject(GO_BANNER, decor_gnomeregan_banners[n].m_positionX, decor_gnomeregan_banners[n].m_positionY, decor_gnomeregan_banners[n].m_positionZ, decor_gnomeregan_banners[n].m_orientation, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0);
+
+                    decor_p4 = true;
+                    break;
+
+                case 5: // INSIDE GNOMEREGAN
+                    if (decor_p5 == true) break;
+                    // 9 Gnomeregan banners
+                    for (int8 n = 23; n < (n + 9); ++n)
+                        me->SummonGameObject(GO_BANNER, decor_gnomeregan_banners[n].m_positionX, decor_gnomeregan_banners[n].m_positionY, decor_gnomeregan_banners[n].m_positionZ, decor_gnomeregan_banners[n].m_orientation, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0);
+
+                    decor_p5 = true;
+                    break;
+
+                case 6: // BOMB PLATFORM INSIDE GNOMEREGAN
+                    if (decor_p6 == true) break;
+
+                    decor_p6 = true;
+                    break;
+                }
+                */
```

Comment:
shall we remove this commented code?

Reply:
it's part of the "decor", once a specific phase of the event finishes - he spawns flags, guards and whatsoever... I have left it so we can use it as a placeholder if we find sniffs in the future.

Reply:
fair enough, maybe just add a comment there explaining what you just mentioned